### PR TITLE
Balance: tweak marauder builds, and remove later game content from them

### PR DESCRIPTION
**Pull request by Hurleveur**
**Balance**

This PR addresses part of a problem described a bit in issue https://github.com/endless-sky/endless-sky/issues/7763

## Summary
The synd shield regen is too easy to acquire from these ships, giving you even less incentive to play checkmate and causing issues if we want to rebalance them, due to how easily and early they're available.
For consistency we also removed the electron guns.

In order to compensate for removing those, I added a smidge more Hai technology to some marauder builds - it could be done more so later on if we feel like this is a good decision. (rn its only 2 small shield gen, one battery and a small a big AM)

Some marauder ships also had a bunch of outfit space left over after the battery buff, and this fills it up, whilst making them and some others a tad bit more original as well.

## Testing Done
n/a

## Save File
n/a

### CODE REVIEW

File: data/human/marauders.txt [hunk 1]
```diff
@@ -533,15 +541,16 @@ ship "Marauder Leviathan" "Marauder Leviathan (Engines)"
...
+		"Heavy Rocket" 80
+		"Gatling Turret" 4
+		"Gatling Gun Ammo" 24000
 		
 		"Fusion Reactor"
 		"LP144a Battery Pack"
-		"D94-YV Shield Generator"
-		"D67-TM Shield Generator"
+		"D94-YV Shield Generator" 2
+		"Supercapacitor" 5
```
Comment (Quantumshark) [inline data/human/marauders.txt:552]:
```suggestion
		"LP036a Battery Pack"
```

Comment (Quantumshark) [reply data/human/marauders.txt:552]: Or, alternatively, since it already has an LP144a battery pack you could replace them with security stations.

Comment (Hurleveur) [reply data/human/marauders.txt:552]:
```suggestion
		"Security Station" 5
```
yeah I like that better

**Comment by Hecter94** [comment]
Note that, if this gets merged, it would be a good idea to look at the availability of Syndicate Regenerators in checkmate and see if they need to be made available earlier now that they're no longer obtainable via Marauders.

Also I think you should mention in the PR description that you're giving Hai tech to some of the marauder ships in lieu of campaign specific unlocks. 
This seems fine to me, but I believe it's worth a mention in the description.

**Comment by Zitchas** [comment]
I wouldn't remove all the synd regenerators. They are supposed to already exist, right? But making them a lot less common is good.

To be honest, I have always found it weird how uncontrolled weapons tech is in human space, and how willing everyone must be to export everything in bulk to everyone.

**Review by ziproot — COMMENTED** [review]
I'm a little confused as to why other changes are being made (such as the engines marauder arrow losing its anti-missile turret) that aren't related to otherwise unobtainable content.

**Comment by Hurleveur** [comment]
Those are just improvements to the builds to make them more original. ig I could stick this only to what it has to be, but we didnt do many of those changes, it's mostly testing the waters.

**Review by ziproot — APPROVED** [review]
Makes sense to me

**Comment by Hurleveur** [comment]
> I wouldn't remove all the synd regenerators. They are supposed to already exist, right? But making them a lot less common is good.

The big issue is that so long as a single one remain before they're sold the argument can be made that they should have X stats, because they're not truly a campaign unlock. So instead of having just a few of them I'd rather have them completely unavailable, and have actual hai regenerators instead, that act very similarly anyway.

**Comment by Quantumshark** [comment]
> I'm a little confused as to why other changes are being made (such as the engines marauder arrow losing its anti-missile turret) that aren't related to otherwise unobtainable content.

The change to the engines marauder arrow was made because it wasn't actually using the extra engine space it had over the regular marauder arrow. I'm also not sure how helpful an anti-missile is on something that fast - in the unlikely case that it does hit a missile, the AM isn't going to get time to shoot that missile down.

**Comment by alexrovw** [comment]
This looks really good and aligns with what I was saying on Discord. I'm going to hold off on approval until I get to playtest to compare these ships to their old models in combat, but I'm very happy to see this change.

**Comment by xX-Dillinger-Xx** [comment]
Is this to address issues for 1st time players? 
The reason I ask, is that I don't recall when I became aware that 'farming' was even a thing, the first time I played. I'm pretty sure it was after I finished the FW storyline.   Now for the repeat player starting a new pilot, this issue becomes irrelevant anyway, because repeat players are just going to go straight to where ever they can get the best tech, then come back to play the campaigns. I would assume that most players just head straight for Hai, or Remnant space ASAP. 
On a side note, you refer to a branch of the FW campaign as _'checkmate'_, and that the current issue discourages players from playing 'checkmate'. I've played a few times and still I'm still not sure where the branch off occurs. Sure, I could go rooting around in the games files but from within the game, new players have no idea there is even a choice to make. You all keep forgetting, you all know the story lines and what happens where and when etc. etc.

**Comment by Zitchas** [comment]
> On a side note, you refer to a branch of the FW campaign as _'checkmate'_, and that the current issue discourages players from playing 'checkmate'. I've played a few times and still I'm still not sure where the branch off occurs. Sure, I could go rooting around in the games files but from within the game, new players have no idea there is even a choice to make. You all keep forgetting, you all know the story lines and what happens where and when etc. etc.

The main break point is when going to pick up the Syndicate defector, and the syndicate shows up demanding you turn them over. If you trust the defector, then you go on to reconciliation. If you trust the Syndicate, you go the checkmate route.

And yes, I agree. A lot more attention should be paid to first-time experience, and a lot less to "how people metagame and/or wiki-walkthrough their umpteenth playthrough."

The point does stand that making this equipment rarer in marauder fleets does make acquiring it in the reconciliation branch a bit more rewarding; but I don't really think it's that major of thing. I also don't feel like super-duper shield regenerators really fits the Syndicate as an overal bonus to siding with them. 

Honestly, I'd more expect them to either fork over a bigger salary (they've got more money than any other human faction, really.) and/or have their ships and outfits available to the player at a steep discount. Call it fleet support or something, for people that want lots of disposable ships rather than solo/few powerful ships. I'm sure as a community we should be able to come up with some rewards that actually caters to the commercial might of the Syndicate rather than just boiling it down to "get X special tech if you pick A, or get Y special tech if you pick B." That might be easier to balance, but it is a lot more boring and unimaginative. The syndicate is known primarily for political power and wealth. What would fit those? (Reduced penalties for pirating ships aka political favors could be another one...)

**Comment by Hurleveur** [comment]
Ideally when we're done revisiting the marauder ships first time players woudnt see any difference, it'd just be that a bit of hai tech is put in there instead of these weapons, and the build are more varied.
In fact it'd hint at smth beyond human space for those that might not have found that out yet

**Comment by xX-Dillinger-Xx** [comment]
I'm going off topic again, so sorry. I'll try to keep it short.

> The main break point is when going to pick up the Syndicate defector, and the syndicate shows up demanding you turn them over. If you trust the defector, then you go on to reconciliation. If you trust the Syndicate, you go the checkmate route.

I can't remember which time I did what, but 1 time I turned over the defector, 1 time I did not. I was unaware this was such an important decision and that it had big implications either way. I did notice the story was different but I really didn't think why or when it changed.  I mentioned this before, there are a few points during the game that the player reaches a fork like this. Should there be some sort of hand holding at these points. Maybe a dialogue. _" They want me to hand over the defector. Let me think, If he is a defector that means ....... But if he's telling the truth that means....."_ I can't recall the details to fill in the ....... but you get the idea. 

> And yes, I agree. A lot more attention should be paid to first-time experience, and a lot less to "how people metagame and/or wiki-walkthrough their umpteenth playthrough."

It's almost like there needs to be 2 different games LOL.  One for first timers and one for repeat players.  AND NO I'm suggesting that, just thinking out loud.

**Comment by ziproot** [comment]
That reminds me, have we done a poll of reconciliation vs checkmate since the first rework?

**Comment by Hecter94** [comment]
> I was unaware this was such an important decision and that it had big implications either way. I did notice the story was different but I really didn't think why or when it changed.
> Should there be some sort of hand holding at these points.

The main difference is that in the reconciliation route, you make a truce with the Republic by rescuing a Syndicate defector and go to war with the Syndicate. 
In checkmate, you keep the Syndicate as an ally by turning over the defector and continuing your war against the Republic. 

The two options you're given are:
`	"I say we rescue the defector, even if it means war with the Syndicate."`
`	"I say we let the Syndicate have the terrorist, and focus on winning the battle against the Republic."`

This spells out quite succinctly what the choice you're making is and what the consequence of that choice will be; I don't see a need for any additional "handholding." 

However, as you mentioned, this is getting off-topic for the PR, so let's keep the conversation relevant to the proposed changes.

**Comment by Hurleveur** [comment]
If this PR doesnt add enough to compensate for the loss of shield gens, I could add another Diamond shield gen to a ship, and a set of hai engines to another

File: data/human/marauders.txt [hunk 2]
```diff
@@ -169,12 +169,12 @@ ship "Marauder Bounder" "Marauder Bounder (Engines)"
 	add attributes
 		"engine capacity" 15
 	outfits
-		"Heavy Laser"
 		"Quad Blaster Turret" 2
 		
 		"Dwarf Core"
 		"LP072a Battery Pack"
-		"S-270 Regenerator"
+		"Supercapacitor" 3
```
Comment (EjoThims) [inline data/human/marauders.txt:176]: I feel like a blaster to fill the gun slot emptied by the heavy laser would be better here than stacked supercaps when it already has another battery.

File: data/human/marauders.txt [hunk 3]
```diff
@@ -533,15 +541,16 @@ ship "Marauder Leviathan" "Marauder Leviathan (Engines)"
 	add attributes
 		"engine capacity" 40
 	outfits
-		"Electron Beam" 4
-		"Quad Blaster Turret" 4
+		"Heavy Rocket Launcher" 4
+		"Heavy Rocket" 80
+		"Gatling Turret" 4
+		"Gatling Gun Ammo" 24000
```
Comment (EjoThims) [inline data/human/marauders.txt:547]: This is an absolutely brutal ammo eater of a build.

I love it.

File: data/human/marauders.txt [hunk 4]
```diff
@@ -563,13 +572,15 @@ ship "Marauder Leviathan" "Marauder Leviathan (Weapons)"
 	add attributes
 		"weapon capacity" 35
 	outfits
-		"Particle Cannon" 4
-		"Heavy Laser Turret" 4
+		"Particle Cannon" 2
+		"Ion Cannon" 2
+		"Heavy Laser Turret" 3
+		"Anti-Missile Turret"
```
Comment (EjoThims) [inline data/human/marauders.txt:578]: I feel like this would make more sense as a Chameleon AM as well.

File: data/human/marauders.txt [hunk 5]
```diff
@@ -321,7 +326,8 @@ ship "Marauder Falcon" "Marauder Falcon (Weapons)"
 		"weapon capacity" 35
 	outfits
 		"Plasma Cannon" 6
-		"Quad Blaster Turret" 4
+		"Heavy Anti-Missile Turret"
```
Comment (EjoThims) [inline data/human/marauders.txt:329]: I feel like this would make more sense as a Chameleon AM.

**Review by EjoThims — COMMENTED** [review]
Small notes on a couple builds, mostly I think if we're switching these to include AMs, we should go with Chameleons instead of the Human ones.

It not only helps show another piece of Hai tech, but -without- including them it ends up seeming like those Ion Cannons are easier to smuggle out.

File: data/human/marauders.txt (hunk 4 above)
Comment (Hurleveur) [reply data/human/marauders.txt:578]: what about a bullfrog here and a chameleon for the other one?

File: data/human/marauders.txt (hunk 2 above)
Comment (Hurleveur) [reply data/human/marauders.txt:176]: I prefer a heavy rocket pod

File: data/human/marauders.txt (hunk 4 above)
Comment (EjoThims) [reply data/human/marauders.txt]: Yea, for some reason I thought both were Heavies :)

File: data/human/marauders.txt (hunk 2 above)
Comment (EjoThims) [reply data/human/marauders.txt]: Brutal. 😁

**Review by EjoThims — APPROVED** [review]

**Comment by xX-Dillinger-Xx** [comment]
I've been keeping quite but I feel I need to point out that you may be solving one issue but creating another. Sorry, Hurleveur and anyone else working on this.
- You call the S-270 and S-970 generators Syndicate generators. 
  - Nowhere in the game are they referred to as Syndicate produced. 
  - I assumed they had something to do with pirates, because they were only found on Marauders.
  - I never made the 'S' connection because Syndicate outfitters sell the same "D" series as the rest of human space. 
  - They are also never installed on any Syndicate ships before or after picking sides. 
- You claim removing them becomes an incentive to join the Syndicate. 
  - If you remove all of them, I'll never know they even exist before I pick sides and in fact may never know, if I choose the Republic.
  - In order to be an incentive, I need to know they exist and I need to know, or at least assume, they are going to be available from Syndicate outfitters. 
- you want to hint at the existence of Hai tech but not unique Syndicate tech. 
  - I don't see the logic. 
  - Won't players just start farming Hai tech instead? If that happens the Syndicate gens become irrelevant, along with some other human tech. There goes the balance again. 

**TL:DR**
I like the idea of hinting to more advanced tech, if the whole goal is to prevent "farming" these generators or _anything else_ too early:
- make them unplunderable until later in the game
- or make them very rare
- ( my preference) or make the Marauder pirates harder to steal from with a higher chance of self destruct. Players should have to work for it.  
- but just don't hide them

**Comment by bene-dictator** [comment]
> You call the S-270 and S-970 generators Syndicate generators.
> Nowhere in the game are they referred to as Syndicate produced.

In the Checkmate Branch, the player is given access to the regenerators and ionic afterburners in human outfitters _on Syndicate planets_, which Katya says is new technology developed by the Syndicate:
`"Also," says Katya, "as their own expression of gratitude, the Syndicate members of Parliament have offered to make some of their new technology, developed during the war, available to us at an affordable price."`

> They are also never installed on any Syndicate ships before or after picking sides.

The Syndicate ships you fight in Alioth after you pick the Reconciliation path and when you fly through Syndicate space to Mutiny in Gienah with Raven are Mk.II Syndicate ships, which have on them Ionic Afterburners and S-series Regenerators.

> I assumed they had something to do with pirates, because they were only found on Marauders.

This is fair, but it makes no sense why the technologically advanced Navy doesn't have this while the apparently galaxy-wide Marauders are willing to hand them over to you as long as you can plunder them. (The Syndicate is known to keep technology hidden from the Republic – for example, the jump drives and the cloaking device.)

**Comment by Hurleveur** [comment]
@xX-Dillinger-Xx let me explain why I opened this PR:
The issue is that balance wise, we want to adjust these generators - they're too good (better than hai shields? wtf, but anyway).
If they're available early on by plunder that means we'll have to make them worse than if they only become available as a story unlock.

Removing all of them is simply easier, and completely resolves the discussion.
Hinting at other alien species existing is more important than at synd tech (which, like you said, it wasn't even doing).

Checkmate being favored is not really a problem for the first playtrough where you don't know what's coming much either way (tho I'm down for hinting at these tech, but that'd be another PR, plz open an issue or smth if you feel like it btw).
The issue is that when you look at what you get both story and outfit wise it isnt worth it to go the checkmate route. I think it should be more rewarding in terms of outfits than it currently is, hence this PR that tweaks marauder builds so these generators can remain ok.
Then, as Hecter said earlier:
> Note that, if this gets merged, it would be a good idea to look at the availability of Syndicate Regenerators in checkmate and see if they need to be made available earlier now that they're no longer obtainable via Marauders.

In which case it'd be more interesting to go checkmate than it currently is (which is, very little).

**Comment by ziproot** [comment]
> The Syndicate ships you fight in Alioth after you pick the Reconciliation path and when you fly through Syndicate space to Mutiny in Gienah with Raven are Mk.II Syndicate ships, which have on them Ionic Afterburners and S-series Regenerators.

Is there a reason why they only use the S-class regenerators in a specific part of Reconciliation?

**Comment by xX-Dillinger-Xx** [comment]
Most of my comments above pertain to new players. For repeat players, very little of anything we are discussing matters because repeat players are just going to go straight for even better tech.

I still think hiding, i.e. removing, them completely,  is a mistake.  
If I'm standing on one side of a raging river and see a pot of gold on the other, I have an incentive to cross the river, even if it **_is_** difficult. If I don't see anything, there is no incentive to cross. 
We are exposed to lots of unobtainable Republic ships, and we can assume that joining them may allow us to purchase them at some point. So, we have an incentive to join the Republic. 
After you remove Synd. gens. we won't be exposed to _anything_ that creates an incentive to join the Syndicate. Even with the "S" gens. currently being exposed, they don't create an incentive, because they aren't clearly identified as Syndicate.
I would consider the simple existence of currently identified but unobtainable items as an incentive to keep looking for a way to obtain them. e.g. side with the Syndicate. 

I go back to this, if the whole goal is to prevent abusive "farming", why not just make "farming" less attractive? If I understand this correctly, the concern is players having access to outfits that are OP for the point of the story they are currently at. How is swapping  the Synd. generators for Hai generators going to solve that? They are almost equal. Even worse, is adding Hai AM, which will be OP, which then makes human AM obsolete too early. This will encourage "farming" for them. Now we're right back where we started.

**Comment by Hecter94** [comment]
> Most of my comments above pertain to new players. For repeat players, very little of anything we are discussing matters because repeat players are just going to go straight for even better tech.

Not necessarily. I, for one, make it a point to play through FW prior to gaining access to any additional tech. I'm sure I'm not alone in this, either. While I do not doubt that some new players and some returning players will go straight to better tech, that doesn't make the point meaningless. 

>  After you remove Synd. gens. we won't be exposed to _anything_ that creates an incentive to join the Syndicate. Even with the "S" gens. currently being exposed, they don't create an incentive, because they aren't clearly identified as Syndicate. I would consider the simple existence of currently identified but unobtainable items as an incentive to keep looking for a way to obtain them. e.g. side with the Syndicate.

The "incentive" for choosing reconciliation or Checkmate isn't the reward you will gain from the respective campaigns. At least for a new player, there's also no indication that a cloaking device will be your reward for reconciliation or that nukes will be your reward for Checkmate, so neither of those factors into the thought process. The choice is made because the player is forced to make a choice, and they have to go with what they feel is right at the point where they make the choice. 

Ideally, the promise of a reward won't factor into the decision, leaving it as a dilemma that needs to be resolved purely based on what the player feels is the correct choice of action. Providing incentives via rewards, therefore, cheapens that decision by relegating it to a mere gameplay choice. 

> If I understand this correctly, the concern is players having access to outfits that are OP for the point of the story they are currently at. 

Not quite; the main concern is ensuring players don't feel penalized for making the "wrong" choice when they choose to follow the Checkmate or reconciliation branch. 
To ensure that there is no objectively wrong choice, the rewards given by the two branches should be relatively equal, which is currently not the case. Nukes are obsoleted relatively quickly and aren't very reliable in combat, nor are they particularly strong, and since the Regenerators aren't limited to just the Checkmate route, it would be entirely possible to find that one of the rewards for the choice you made is something you already have access to, which makes that reward feel like not a true reward. 

Compared to the cloaking device, it seems reasonable to conclude that Checkmate could use a bit more going for it and making one of its current rewards an exclusive reward could be an option to assist with that. 

> How is swapping the Synd. generators for Hai generators going to solve that? They are almost equal. 

Syndicate generators are decently better than the Hai generators. With the S-970 generating 3.73 shields per outfit space and the Hai Diamond Regenerator only generating 3.06 shields per outfit space, the S-970 is approximately 22% more space efficient, which causes mechanical and thematic issues that I won't get into here. Swapping them would, therefore, allow earlier access to better shield generators while keeping the even better shield generators as a unique campaign-specific unlock. 

> Even worse, is adding Hai AM, which will be OP, which then makes human AM obsolete too early. This will encourage "farming" for them. Now we're right back where we started.

Hai AM is generally equivalent to, or slightly worse than, Human AM. 

<details>
<summary>See this chart, which plots their effectiveness (in average expected kills per second) against each missile type: </summary>

image
</details>

The Heavy Anti-Missile Turret is consistently better than the Chameleon Anti-Missile against almost every missile type while being substantially better against the weak missiles in the Free Worlds campaign. 
Meanwhile, the Bullfrog is roughly equivalent to the Anti-Missile Turret against most missile types, with the Anti-Missile turret pulling ahead against weaker missiles again. 

Of course, the Chameleon and the Bullfrog are slightly smaller than their human counterparts. However, for the Heavy Anti-Missile, at least against most human missiles, even if you adjust for size, it still works out to be more efficient per unit of space, and both turrets are more efficient usages of a hardpoint. 

Effectively, they are about equivalent, with the Heavy Anti-Missile Turret being slightly better than the Chameleon against weak missiles but otherwise trading space efficiency for hardpoint efficiency, and the Anti-Missile Turret likewise trading space efficiency for hardpoint efficiency compared to the Bullfrog. 

Overall, I certainly wouldn't say that one obsoletes the other or that one is substantially more powerful than the other.

**Comment by Hurleveur** [comment]
I dont see how the argument can be made that this would point towards the synd having it much (an s in the name? that means nothing)
That's smth thats missing and should be added separately.

**Review by Quantumshark — APPROVED** [review]

**Comment by brendanjones** [comment]
Fwiw I had always assumed the S-class shield generators were of Hai origin, because they look very similar to Hai shield generators. Is that intentional? If not it might be good to adjust their image.

I've also always thought it odd that they're better than Hai generators - I think their generation should be dropped 10-20% so that they're still far better than other human shield tech but a little worse than the Hai tech. Which gives players a juicy reward for finding their way out of Human space!

Funnily enough I had no idea that they were Syndicate tech that you get from some split in the mission string, I've somehow missed that entirely despite playing through many times.

**Comment by xX-Dillinger-Xx** [comment]
@brendanjones 
> Funnily enough I had no idea that they were Syndicate tech that you get from some split in the mission string, I've somehow missed that entirely despite playing through many times.

Your not the only one. That's why I suggested to change their description.
